### DISCUSSION TITLE
م

### DISCUSSION THREAD

**Discussion by Lida5** [community/community#140733] [New to GitHub]
مرحبا

**Comment by milad-2003** [comment]
Well done!

**Comment by ghostinhershell** [comment]
Hi @Lida5, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
